lib/awsutil/accounts.go

```diff
@@ -79,6 +80,8 @@ func listFile(pathname string, identifierKeyName string) ([]string, error) {
 			accountNames = append(accountNames, lastAccountName)
 			accessKeyIds[value] = struct{}{}
 			lastAccountName = ""
+		} else {
+			return nil, fmt.Errorf("Duplicate key identifier found: '%s'", line)
```

Comment:
This will break some existing deployments which have duplicate keys. This is why I originally added this test and skipped.

Reply:
fair enough.